acme/src/main/java/io/micronaut/acme/services/AcmeService.java

```diff
@@ -143,10 +143,10 @@ public X509Certificate getCurrentCertificate() {
      *
      * @param domains List of domains to order a certificate for
      * @throws AcmeException if any issues occur during ordering of certificate
+     *
+     * @return order for the given list of domains
      */
-    public void orderCertificate(List<String> domains) throws AcmeException {
-        AtomicInteger orderRetryAttempts = new AtomicInteger(acmeConfiguration.getOrder().getRefreshAttempts());
-
+    public Order orderCertificate(List<String> domains) throws AcmeException {
```

Comment:
This is a breaking change. The original method signature must remain intact. You can create a new method for this purpose